README.md

```diff
@@ -1,2 +1,43 @@
-# reviews-service
-Repo for the Airbnb reviews component
+# Air Tree n Tree Reviews Service
+
+## API spec
+`GET /api/reviews/:roomid/?review_group=[selected pagination number]`
```

Comment:
Would be nice if there were type specifications for the params as well.

| param | type |
| ------ | ---| ---
|page | Number |